src/Instrumentation.ts

```diff
@@ -29,6 +29,7 @@ export class Instrumentation {
         const w = typeof window !== 'undefined' ? window : undefined;
         const { performance } = (g || w || {performance: undefined});
 
+        // @ts-ignore
         if (performance && performance.mark && performance.measure) {
```

Comment:
ts-ignore shouldn't be needed here. double-inversion will make this a boolean. I think TS thinks that this should always be defined, but having the safety check is valid
```suggestion
        if (performance && !!performance.mark && !!performance.measure) {
```